numba/_dispatcher.c

```diff
@@ -6,6 +6,83 @@
 
 #include "_dispatcher.h"
 #include "_typeof.h"
+#include "frameobject.h"
+
+#define PROFILE_CALL 1
```

Comment:
I'm not sure why a macro is necessary, but if it is, you should add a one-line description.